src/logics/LALogic.h

```diff
@@ -57,8 +58,11 @@ class LALogic: public Logic
     virtual PTRef     mkConst         (const opensmt::Number& c);
     virtual PTRef     mkConst         (const char* num);
     virtual PTRef     mkNumVar        (const char* name);
+    PTRef   mkIte                     (vec<PTRef>&) override;
+    PTRef   mkIte                     (PTRef cond, PTRef true_tr, PTRef false_tr) override { vec<PTRef> tmp{cond, true_tr, false_tr}; return mkIte(tmp); }
```

Comment:
This second override is doing the same thing as the base class version. Consider removing it?